huawei_lte_api/api/Vpn.py

```diff
@@ -19,12 +20,14 @@ def l2tp_settings(self) -> GetResponseType:
     def pptp_settings(self) -> GetResponseType:
         return self._session.get('vpn/pptp_settings')
 
-    def toggle_status(self, enable: bool = True) -> SetResponseType:
+    def toggle_status(self, vpn_type: VPNType = VPNType.PPTP, enable: bool = True) -> SetResponseType:
         data = {
             "enable": "1" if enable else "0",
         }
+
         return self._session.post_set(
-            "vpn/l2tp_settings",
+            # pptp_settings | l2tp_settings
+            f"vpn/{vpn_type.value}_settings",
```

Comment:
Please don't use f strings, they are not supported(well) in older versions of python use:
```
'vpn/{vpn_type}_settings.format(vpn_type=vpn_type.value),
```
Thank you